ci/kind/kind-setup.sh

```diff
@@ -580,7 +580,9 @@ function clean_kind {
               echo "$name $creationTimestamp" >> ~/.antrea/.clusters.swp
           fi
       done < ~/.antrea/.clusters
-      mv ~/.antrea/.clusters.swp ~/.antrea/.clusters
+      if [[ -e ~/.antrea/.clusters.swp ]]; then
+          mv ~/.antrea/.clusters.swp ~/.antrea/.clusters
+      fi
```

Comment:
this is incorrect IMO
if `~/.antrea/.clusters.swp` does not exit, then we should end up with an empty `~/.antrea/.clusters`, and this is not the case here

the correct fix is to add `: > ~/.antrea/.clusters.swp` before the `while` loop, in order to make that the file exists and is empty

it would have been better to have this fix in a separate PR too, but I guess not a big deal in this specific case